src/variable.jl

```diff
@@ -47,6 +47,7 @@ end
 function set_column_name(m::Optimizer, vi::MOI.VariableIndex, name::String)
     set_column_name(m.task, mosek_index(m, vi), name)
 end
+MosekTools.set_column_name(::Mosek.MSKtask, ::MosekTools.MatrixIndex, name::String) = name    
```

Comment:
This should throw an error.

Reply:
Okay, let me test what works, and I will create a new pull request. I am closing this one. Thanks @blegat !